scripts/required_test.js

```diff
@@ -37,6 +37,7 @@ require("../test/unique_count_sorted.test.js");
 require("../test/standard_deviation.test.js");
 require("../test/standard_normal_table.js");
 require("../test/sum.test.js");
+require("../test/combinationsReplacement.js");
```

Comment:
Could this file be named `combinations_replacement.js`? Unfortunately with the intersection of OSX's case insensitivity and git's case sensitivity, uppercase characters in filenames can be very difficult to work with.